beamer/tests/contracts/test_request_manager.py

```diff
@@ -1436,3 +1436,29 @@ def test_transfer_limit_requests(request_manager, token):
     make_request(request_manager, token, requester, requester, transfer_limit + 1)
 
     assert token.balanceOf(requester) == 0
+
+
+def test_withdraw_invalid_fill(
```

Comment:
I would suggest to change the name to something like `test_withdraw_invalid_fill_no_challenger`